src/index.ts

```diff
@@ -20,12 +20,15 @@ export async function run(): Promise<void> {
     const tagsList: string[] = tags.split(" ");
     const newImage = `${image}:${tagsList[0]}`;
     const useOCI = core.getInput(Inputs.OCI) === "true";
+    let archs = core.getInput(Inputs.ARCHS);
+    // remove white spaces (if any) in archs input
+    archs = archs.replace(/\s+/g, "");
```

Comment:
this looks to me like spaces are allowed after the comma because they will get removed here anyway

Reply:
Yes, spaces are not allowed.

Reply:
To clarify, above I was asking if the user could input spaces. I see that spaces are not allowed in the argument actually passed to buildah.

Reply:
Yes, spaces are not allowed in the argument actually passed to buildah, to make it user friendly if by mistake user added any extra space then that will get removed.